TenantFile/Api.UnitTests/.firebaserc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Let's pull out the firebase stuff and use the simulators for the tenant-file folder.